nucleus/common/common-util/src/main/java/com/sun/enterprise/security/store/AsadminSecurityUtil.java

```diff
@@ -210,15 +210,15 @@ public AsadminTruststore getAsadminTruststore() {
         try {
             return openTruststore(password);
         } catch (CertificateException | KeyStoreException | NoSuchAlgorithmException | IOException e) {
-            throw new RuntimeException(e);
+            logger.log(Level.WARNING, String.format("Error when opening truststore: %s", e));
         }
     }
 
     public KeyStore getAsadminKeystore() {
         try {
             return openKeystore(password);
         } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException e) {
-            throw new RuntimeException(e);
+            logger.log(Level.WARNING, String.format("Error when opening keystore: %s", e));
```

Comment:
Sorry for being confusing. The exception is actually ok to throw IMHO. I just asked you to add some context information in form of a text. Kind of the text you have logged now.